code/game/objects/items/devices/scanners.dm

```diff
@@ -478,11 +491,18 @@ REAGENT SCANNER
 					recent_fail = 1
 		if(dat)
 			to_chat(user, "<span class='notice'>Chemicals found: [dat]</span>")
+			if(alert("Do you want to print a log file?","Scan Results","Yes", "No") == "Yes")

```

Comment:
What happens if someone drops the scanner in a disposals bin and waits for it to be halfway across the station before saying "yes"?